src/playwright/transformPlaywrightJson.ts

```diff
@@ -43,7 +43,7 @@ export const transformPlaywrightJson = (src: string) => {
   }
   const stories = Object.values(json.stories) as Story[];
   const titleIdToStories = stories.reduce((acc, story) => {
-    const titleId = toId(story.title);
+    const titleId = toId(story.title, story.name);
```

Comment:
After giving a bit more thought I don't think this is right. The idea is to actually group stories per component.
So `toId(story.title)` would generate an id of e.g. `example-page` which would result in an aggregated object like:

```js
{
  "example-page": `
    describe("Example/Page", () => {
      describe("Logged In", () => {
        it("test", async () => {
          // ...
        })
      })
      describe("Logged Out", () => {
        it("test", async () => {
          // ...
        })
      })
    })
  `
}
```


And `toId(story.title, story.name)` would generate the entire title like `example-header--logged-in`, so the aggregation would not be correct, as we'd have a lot of describes with the same name
```js
{
  "example-page--logged-in": `
    describe("Example/Page", () => {
      describe("Logged In", () => {
        it("test", async () => {
          // ...
        })
      })
    })
  `,
  "example-page--logged-out": `
    describe("Example/Page", () => {
      describe("Logged Out", () => {
        it("test", async () => {
          // ...
        })
      })
    })
  `,
}
```

Reply:
The newest version of the toID method defines the second parameter as optional (https://github.com/ComponentDriven/csf/blob/b02ef28ede03446514f64dd10085c9cef7de5d7d/src/index.ts#L31) but I cannot find a released version of it

Reply:
Change is done!